### PR TITLE
Fix/point to latest versions

### PR DESCRIPTION
**Pull request by ThetaSinner**
### Summary

You can confirm here that this fix is equivalent to the fix we did for 0_3 https://github.com/holochain/holochain/blob/develop-0.3/.github/workflows/holochain-build-and-test.yml#L34

### TODO:
- [ ] CHANGELOGs updated with appropriate info
- [ ] All code changes are reflected in docs, including module-level docs

### CODE REVIEW

File: .github/workflows/holochain-build-and-test.yml [hunk 1]
```diff
@@ -30,18 +30,8 @@ jobs:
...
-          - pkgs:
-              - build-holonix-tests-integration
-            extra_arg: "--override-input versions ./versions/0_2"
-          - pkgs:
-              - build-holonix-tests-integration
-            extra_arg: "--override-input versions ./versions/0_3"
-          - pkgs:
-              - build-holonix-tests-integration
-            extra_arg: "--override-input versions ./versions/weekly"
+            # TODO switch to `0_4` when that it is released
```
Comment (jost-s) [inline .github/workflows/holochain-build-and-test.yml:33]:
```suggestion
            # TODO switch to `0_4` when that is released
```

File: .github/workflows/update-dispatch.yml [hunk 2]
```diff
@@ -33,8 +33,6 @@ jobs:
           - name: ${{ github.ref_name }}
             skip: true
         update:
-          - source: "versions/weekly"
-            cmd: "nix run .#scripts-repo-flake-update weekly"
```
Comment (jost-s) [inline .github/workflows/update-dispatch.yml:37]: Why is this removed?

Comment (ThetaSinner) [reply .github/workflows/update-dispatch.yml:37]: We don't maintain the `versions` directory on release branches, it's all done from develop. The root of this whole problem is that we want to point at local paths on `main` where we are maintaining this stuff so we see changes in PRs but from release branches we don't want that. So each time we branch, I slowly find all the misjudged references to `./` and replace them with `github:holochain/holochain` :(

Comment (jost-s) [reply .github/workflows/update-dispatch.yml:37]: Mmh, that seems like the answer if I had asked why you replaced `./versions/weekly` with `github:holochain/holochain?dir=versions/weekly` above. :thinking:

**Review by jost-s — APPROVED** [review]

File: .github/workflows/update-dispatch.yml (hunk 2 above)
Comment (ThetaSinner) [reply .github/workflows/update-dispatch.yml:37]: Right yeah, so what this workflow does is update the inputs. E.g. if a new scaffolding had released for `weekly` and it will correctly update `./versions/weekly` but then everyone will read the version from `develop`, not the one you updated on `develop-0.4`. So it's confusing to even update it IMO

Comment (jost-s) [reply .github/workflows/update-dispatch.yml:37]: Yes, got it.
